src/app/games/mancala/ba-awa/BaAwaTutorial.ts

```diff
@@ -0,0 +1,88 @@
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { BaAwaConfig } from './BaAwaConfig';
+import { BaAwaRules } from './BaAwaRules';
+import { MancalaTutorial } from '../common/MancalaTutorial';
+import { MancalaState } from '../common/MancalaState';
+import { MancalaDistribution, MancalaMove } from '../common/MancalaMove';
+import { TutorialStepFailure } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStepFailure';
+
+const defaultConfig: MGPOptional<BaAwaConfig> = BaAwaRules.get().getDefaultRulesConfig();
+
+export class BaAwaTutorial extends Tutorial {
+
+    public gameName: string = $localize`Ba-awa`;
+
+    public tutorial: TutorialStep[] = [
+        MancalaTutorial.intro(
+            this.gameName,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        TutorialStep.informational(
+            $localize`Ba-awa`,
+            $localize`Bonus fact: Ba-awa, also called Adi, is played mainly in Ghana.`,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        MancalaTutorial.sowing(
+            new MancalaState([
+                [4, 4, 4, 4, 4, 4],
+                [4, 0, 4, 4, 4, 4],
+            ], 0, [0, 0]),
+        ),
+        TutorialStep.fromMove(
+            $localize`Multiple laps sowing`,
+            $localize`However, the distribution only stops when the last seed is dropped in a house that contains zero or three seeds (before drop).<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 2, 4, 0, 0, 0],
+                [0, 0, 0, 5, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(3))],
+            $localize`So, after landing in the house with 3 seeds, a second lap has been started`,
+            TutorialStepFailure.FAILED_TRY_AGAIN(),
+        ),
+        TutorialStep.fromMove(
+            $localize`Captures during distribution (1/2)`,
+            $localize`If, during some distribution, you pass by one of your house that contains 3 seeds, and drop a fourth seed, you capture the house immediately, then continue the distribution!<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 8, 0, 0, 0, 0],
+                [0, 0, 3, 2, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(3))],
+            $localize`Congratulations, you captured 4 seeds.`,
+            MancalaTutorial.YOU_DID_NOT_CAPTURE_ANY_SEEDS(),
+        ),
+        TutorialStep.fromMove(
+            $localize`Captures during distribution (2/2)`,
+            $localize`If, during some distribution, you pass by one house of the opponent that contains 3 seeds, and drop a fourth seed, the opponent capture the house immediately (while you continue to distribute)!<br/><br/>You're playing Dark, do such a move!`,
```

Comment:
the opponent *captures* [...], while you continue to distribute!

(pas grand fan des parenthèses, surtout si c'est pour terminer par du parenthèse-ponctuation)

Reply:
j'avoue !